meshtastic/mesh.proto

```diff
@@ -417,6 +417,11 @@ enum HardwareModel {
    */
   T_WATCH_S3 = 51;
 
+  /*
+   * B&Q Consulting Nano G2 Ultra: https://wiki.uniteng.com/en/meshtastic/nano-g2-ultra
+   */
+  NANO_G2_ULTRA = 52;
```

Comment:
18 is free, right after 17 ... why not use that one? :-)

Reply:
Oh, let me resubmit a PR :)